src/Simplex/Chat/Library/Commands.hs

```diff
@@ -3369,7 +3372,7 @@ processChatCommand' vr = \case
           Nothing -> Just GRAuthor
           Just (GCSMemberSupport Nothing)
             | memberPending membership -> Nothing
```

Comment:
why this returns Nothing and not GRObserver?

Reply:
if it works observer should be ok

Reply:
I don't know if it works